tutor/src/components/invalid-page.cjsx

```diff
@@ -1,13 +1,18 @@
 React  = require 'react'
 {Link} = require 'react-router'
+BS = require 'react-bootstrap'
```

Comment:
Eeek, so this has been wrong for awhile huh?

Why not use TutorLink, then we wouldn't need the `TutorRouter.makePathname` below